Upgrade commons-lang3 from 3.9 to 3.11

sebthom opened the pull request:
Version 3.11 is backwards compatible. For changes see https://commons.apache.org/proper/commons-lang/changes-report.html

I am proposing this change because using the quarkus-bom (at least in our case) results in a version downgrade of the commons-lang transitive dependency of other (non-quarkus) project dependencies.

gsmet commented:
Thanks! Could you also add it there: https://github.com/quarkusio/quarkus/blob/master/.github/dependabot.yml ?

sebthom committed:
Upgrade commons-lang3 from 3.9 to 3.11

Version 3.11 is backwards compatible. For changes see https://commons.apache.org/proper/commons-lang/changes-report.html

I am proposing this change because using the quarkus-bom (at least in our case) results in a version downgrade of the commons-lang transitive dependency of other (non-quarkus) project dependencies.

sebthom commented:
> 
> Thanks! Could you also add it there: [https://github.com/quarkusio/quarkus/blob/master/.github/dependabot.yml](https://github.com/quarkusio/quarkus/blob/master/.github/dependabot.yml?rgh-link-date=2021-01-21T16%3A49%3A19Z) ?

Done.

gsmet commented:
Perfect, thanks!
